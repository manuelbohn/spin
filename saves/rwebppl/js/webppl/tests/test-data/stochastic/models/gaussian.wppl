var model = function() {
  return gaussian(3, 2);
};
