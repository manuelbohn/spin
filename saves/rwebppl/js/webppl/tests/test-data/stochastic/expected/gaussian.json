{
  "mean": 3,
  "std": 2
}
